source/unified-test-format/tests/invalid/do.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Not sure I understand what this file is for. Is this to be used during development? If so, could you add a comment or other way to explain its usage? Also, while I appreciate building the popularity in PHP, I'm not sure if it's going to be the most portable, as I don't expect everybody to have PHP ready (especially with MacOS not shipping PHP by default in the near future).

Reply:
Oops, I didn't mean to add this.

I used this early on to sync descriptions in each file with the filename.